package.json

```
@@ -59,6 +60,23 @@
   "publishConfig": {
     "access": "public"
   },
+  "release": {
+    "plugins": [
```

Let's be explicit about which branches we release from 

```suggestion
    "branches": ["master", "beta"],
    "plugins": [
```